docs/app/Examples/elements/Header/Content/HeaderSubheaderExample.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import {Header} from 'stardust';
+
+export default class HeaderSubheaderExample extends Component {
+  render() {
+    return (
+      <Header.H2>
+        Account Settings
+        <div className='sub header'>
+          Manage your account settings and set email preferences
+        </div>
```

Comment:
I wondered that actually. If you do `Header.Sub` it will "work" but it capitalizes everything.